client_option_progress_cheggaaa.go

```diff
@@ -0,0 +1,67 @@
+package getter
+
+import (
+	"io"
+	"path/filepath"
+
+	pb "gopkg.in/cheggaaa/pb.v1"
```

Comment:
I think this is an issue for me in that that if we were to roll this progress bar in with go-getter, that would mean that _all_ systems that depend on go-getter would pull this in, even if they wanted to implement their own progress bar. I would prefer that this be removed, we keep the interface, and progress bars be explicitly implemented within each consuming implementation.